src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function add(x) {
+    if (x === undefined) {
```

Comment:
```suggestion
    if (!x) {
```

Reply:
It will not work with zero as an argument.